.github/workflows/test.yml

```diff
@@ -16,7 +16,7 @@ jobs:
           ref: ${{ github.event.pull_request.head.sha }}
           fetch-depth: 0
       - uses: actions/setup-python@v2
-      - run: pip install towncrier==19.2.0
+      - run: pip install 'towncrier>=22,<23'
```

Comment:
do we need to pin

Reply:
Like we discussed, Synapse doesn't pin (it uses `towncrier>=18.6.0rc1`), so there's precedent in not having an upper bound & `>=22` is probably fine.